pom.xml

```diff
@@ -72,7 +72,7 @@
     <bigquery.connector.hadoop2.version>hadoop2-1.0.0</bigquery.connector.hadoop2.version>
     <commons.codec.version>1.4</commons.codec.version>
     <cdap.version>6.3.0-SNAPSHOT</cdap.version>
-    <cdap.plugin.version>2.4.0-SNAPSHOT</cdap.plugin.version>
+    <cdap.plugin.version>2.5.0-SNAPSHOT</cdap.plugin.version>
```

Comment:
What should the pom version for this be? live develop has 2.5.0-SNAPSHOT or should we ignore pom versions if they are different?